Common/Securities/IBuyingPowerModel.cs

```diff
@@ -37,6 +37,26 @@ public interface IBuyingPowerModel
         /// <param name="leverage">The new leverage</param>
         void SetLeverage(Security security, decimal leverage);
 
+        /// <summary>
+        /// Gets the margin currently allocated to the specified holding
+        /// </summary>
+        /// <param name="parameters">An object containing the security</param>
+        /// <returns>The maintenance margin required for the </returns>
```

Comment:
nit - probably a copy pasta but `The maintenance margin required for the`...`provided quantity`?